### ISSUE TITLE
TM task comment box "Notify project manager of this comment" feature request

### ISSUE THREAD

**Issue by bgirardot** [hotosm/osm-tasking-manager2#839]
It would be really helpful (i hope) if there was an obvious checkbox when making a comment on a task square that says "notify project manager of this comment"

I realize the @name option exists, but that is a bit more advanced and means that the person making the comment has to leave that tab to find the project manager. And they might not even realize there is a project manager.

On the backend it could just auto add @projectmanagername to the comment so the regular code works to get it delivered. Or now that I think about it, it could just be a button that puts the @projectmanagername into the comment field directly so it is all client side and the back end code has to do nothing different.

I note that this is in addition to the tasking manager comment feed I proposed as well. Project managers might not keep up on the stream, although I hope they do. hopefully the checkbox and direct TM message will ensure the project manager gets the message.

(the new and improved notification of TM messages is quite helpful by the way)

**Comment by aawiseman** [comment]
Agreed, this was mentioned as a useful feature at the Missing Maps meeting in Brussels
